modules/apps/feature-flag/feature-flag-api/src/main/java/com/liferay/feature/flag/manager/FeatureFlagPreferencesManager.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.feature.flag.manager;
+
+/**
+ * @author Thiago Buarque
+ */
+public interface FeatureFlagPreferencesManager {
```

Comment:
Does this need to be part of the API? I think this could stay internal to the `impl` module, right?